sp/src/game/server/hl2/npc_combine.h

```diff
@@ -42,6 +42,8 @@
 #define SF_COMBINE_COMMANDABLE ( 1 << 19 ) // Added by 1upD - soldier commandable spawnflag
 #define SF_COMBINE_REGENERATE ( 1 << 20 ) // Added by 1upD - soldier health regen spawnflag
 #define SF_COMBINE_NO_MANHACK_DEPLOY ( 1 << 21 ) // Blixibon -- Manhack toss spawnflag
+#define SF_COMBINE_MEDIC ( 1 << 22 ) // Citizen-like medic
```

Comment:
Unless you change your mind, I think we want to make this a key value so we're not worried about the order of spawnflags or if another spawnflag could come from Mapbase and collide or something along those lines